games-engines/frotz/frotz-2.50.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A Portable Z-Machine Interpreter"
+DESCRIPTION="Frotz is an interpreter for Infocom games \
+and other Z-machine games. \
+It complies with standard 1.0 of Graham Nelson's specification. \
+It was written by Stefan Jokisch in 1995-1997"
+HOMEPAGE="https://davidgriffith.gitlab.io/frotz/"
+COPYRIGHT="1995-1997 Stefan Jokish
+		1997-2019 David Griffith"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://gitlab.com/DavidGriffith/frotz/-/archive/$portVersion/frotz-$portVersion.tar.bz2"
+CHECKSUM_SHA256="7fd1185318dec2c897228d6ed81d4d2599d64d1e5bb7f913c09d17f3337ec663"
+PATCHES="frotz-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	frotz$secondaryArchSuffix = $portVersion
+	cmd:frotz
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+#	lib:libao$secondaryArchSuffix
```

Comment:
You can enable libao, seems the upstream patch for Haiku hasn't been merged, but it's included in our own patchset for libao

Reply:
right